laplace/baselaplace.py

```diff
@@ -336,34 +331,37 @@ def __init__(self, model, likelihood, sigma_noise=1., prior_precision=1.,
             'GGN or EF backends required in ParametricLaplace.'
         super().__init__(model, likelihood, sigma_noise, prior_precision,
                          prior_mean, temperature, backend, backend_kwargs)
-
-        self.H = None
-
+        try:
+            self._init_H()
+        except AttributeError:  # necessary information not yet available
+            pass
+        self.loss = 0
+        self.n_data = 0
```

Comment:
`loss` and `n_data` attributes are already initialized in the `BaseLaplace` class.